### PR TITLE
DRUP-1156: #ready adds form label on search box on /locations

### PR DESCRIPTION
**Pull request by z3cka**

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_search/uclalib_search.module [hunk 1]
```diff
@@ -30,9 +30,13 @@ function uclalib_search_entity_property_info_alter(&$info) {
...
  */
 function uclalib_search_form_views_exposed_form_alter(&$form, &$form_state, $form_id) {
-  if ($form['#id'] == 'views-exposed-form-staff-directory-staff-directory-pane' || $form['#id'] == 'views-exposed-form-news-search-news-search') {
-    $form['search_api_views_fulltext']['#title'] = t('Search');
-    $form['search_api_views_fulltext']['#title_display'] = 'invisible';
+  if (
+      $form['#id'] == 'views-exposed-form-staff-directory-staff-directory-pane' || 
+      $form['#id'] == 'views-exposed-form-news-search-news-search' ||
+      $form['#id'] == 'views-exposed-form-location-search-panel-pane-1'
+    ) {
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: Not sure if this is a good way to format an `if` condition with three `OR`s ... thoughts?

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: I was thinking it add readability.

Comment (emb03) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: @z3cka looks fine, there is `in_array` also:`if (in_array($var, array("abc", "def", "ghi")))`

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: thanks @emb03, I do use that if the list gets really long.

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: I think I'm going to make a small change based on: http://stackoverflow.com/a/31717900/2345114

File: www/sites/all/modules/features/uclalib_search/uclalib_search.module [hunk 2]
```diff
@@ -30,9 +30,12 @@ function uclalib_search_entity_property_info_alter(&$info) {
  * Implements hook_form_FORM_ID_alter() for FORM_ID().
  */
 function uclalib_search_form_views_exposed_form_alter(&$form, &$form_state, $form_id) {
-  if ($form['#id'] == 'views-exposed-form-staff-directory-staff-directory-pane' || $form['#id'] == 'views-exposed-form-news-search-news-search') {
-    $form['search_api_views_fulltext']['#title'] = t('Search');
-    $form['search_api_views_fulltext']['#title_display'] = 'invisible';
+  if ($form['#id'] == 'views-exposed-form-staff-directory-staff-directory-pane'
+      || $form['#id'] == 'views-exposed-form-news-search-news-search' ||
+      || $form['#id'] == 'views-exposed-form-location-search-panel-pane-1'
+    ) {
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_search/uclalib_search.module]: I think I like this format the best, got it from: http://stackoverflow.com/a/31717900/2345114

**Comment by z3cka** [comment]
deploying now to `www-test` along with #397

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.` seems to be passing QA now

**Pull request merged**